modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/scope/Scope.es.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import getCN from 'classnames';
+import {LearnMessage} from 'frontend-js-components-web';
+import React from 'react';
+
+import {
+	PORTAL_TOOLTIP_TRIGGER_CLASS,
+	SCOPE_TYPES,
+} from '../../utils/constants.es';
+import SelectScope from './SelectScope.es';
+
+export default function Scope({
+	disabled,
+	initialGroupExternalReferenceCode,
+	initialSxpBlueprintExternalReferenceCode,
+	onChange,
+	onResultsUpdate,
+	selectedType,
+	touched,
+}) {
+	const _handleScopeTypeChange = (type) =>
+		onChange({
+			touched: false,
+			type,
+			value: '',
+		});
+
+	const _renderRadioButton = (value, helptext, label) => (
+		<div className="custom-control custom-radio">
+			<label>
+				<input
+					checked={value === selectedType}
+					className="custom-control-input"
+					disabled={disabled}
+					onChange={() => _handleScopeTypeChange(value)}
+					role="radio"
+					type="radio"
+				/>
+
+				<span className="custom-control-label">
+					<span className="custom-control-label-text">
+						{label}
+
+						<span
+							className={getCN(
+								'input-label-help-icon',
+								'text-secondary',
+								PORTAL_TOOLTIP_TRIGGER_CLASS
+							)}
+							data-title={helptext}
+						>
+							<ClayIcon symbol="question-circle-full" />
+						</span>
+					</span>
+				</span>
+			</label>
+		</div>
+	);
+
+	return (
+		<div className="results-rankings-scope-selector">
+			{_renderRadioButton(
+				SCOPE_TYPES.EVERYWHERE,
+				Liferay.Language.get('result-rankings-scope-everything-help'),
+				Liferay.Language.get('everything')
+			)}
+
+			{Liferay.FeatureFlags['LPS-157988'] && (
+				<>
+					{_renderRadioButton(
+						SCOPE_TYPES.SITE,
+						Liferay.Language.get('result-rankings-scope-site-help'),
+						Liferay.Language.get('site')
+					)}
+
+					<SelectScope
+						disabled={disabled}
+						fetchByURL="/o/headless-admin-user/v1.0/sites/by-external-reference-code/"
+						fetchURL="/o/headless-admin-user/v1.0/sites"
```

Comment:
Can we rename these to something more specific? It's very vague what's these do.